src/System.Management.Automation/System.Management.Automation.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
```

Comment:
Did you change the encoding of the file?

Reply:
BOM removed.